Qt/mainwindow.h

```diff
@@ -244,7 +244,7 @@ class MenuAction : public QAction
 public:
 	// Add to QMenu
 	MenuAction(QWidget* parent, const char *callback, const char *text, QKeySequence key = 0) :
-		QAction(parent), _text(text), _eventCheck(0), _eventUncheck(0), _stateEnable(-1), _stateDisable(-1), _enableStepping(false)
+		QAction(parent), _text(text), _eventCheck(0), _eventUncheck(0), _stateEnable(-1), _stateDisable(-1), _enabledFunc(nullptr)
```

Comment:
Hm, `_enableStepping` still exists and is still initialized by the other constructor - should we replace that with a func too?

Would also prefer to move these to initialize with the properties themselves now, since we're changing these lines anyway.

-[Unknown]

Reply:
Mhh I was really sure I removed it, it's not used anywhere so no need to replace.